components/brave_ads/core/internal/common/url/url_request_string_util.cc

```diff
@@ -30,46 +29,42 @@ bool ShouldAllowHeader(const std::string& header) {
 
 std::string HeadersToString(const std::vector<std::string>& headers,
                             const int indent = 4) {
-  std::vector<std::string> formatted_headers;
+  std::ostringstream formatted_headers;
```

Comment:
I'm not sure that streams should be used for such things. `the libraries in absl/strings or the equivalent are usually a better choice than std::stringstream.`

https://google.github.io/styleguide/cppguide.html#Streams

Reply:
Reverted the stream changes, thanks